Source/Fuse.Triggers/Trigger.uno

```diff
@@ -16,6 +16,11 @@ namespace Fuse.Triggers
 		Never,
 		/** Only changes during the rooting frame are handled as bypass, without special exceptions. */
 		Rooting,
+		
+		/** Deprecated: 2017-07-21
+			For possible backwards compatibilty, like Standard but excludes the check for layout bypass.
+			This mode should not be used. */
+		ExceptLayout,
```

Comment:
Hmm, it does, but then using it inside Trigger also creates warnings:

`Source/Fuse.Triggers/Trigger.uno(109.36): W4139: Fuse.Triggers.TriggerBypassMode.ExceptLayout is obsolete`

How could we avoid those?